pkgs/development/python-modules/dparse/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, six
+, pyyaml
+, packaging
+}:
+
+buildPythonPackage rec {
+  pname = "dparse";
+  version = "0.4.1";
+
+  buildInputs = [ six pyyaml packaging ];
+
+  # Inpure integration tests
+  doCheck = false;
```

Comment:
you can ignore specific tests with pytest. It's still good to include unit tests:
```
  checkPhase = ''
    pytest --ignore=some_file.py -k 'not bad_test'
```